service/logger.py

```diff
@@ -0,0 +1,33 @@
+import logging
+from paste.translogger import TransLogger
+
+
+def get_stdout_handler():
+    format_string = '%(asctime)s - %(name)s - %(levelname)s - %(message)s'
+    stdout_handler = logging.StreamHandler()
+    stdout_handler.setFormatter(logging.Formatter(format_string))
+    return stdout_handler
+
+
+def init_logger(logger_name, loglevel):
+    logger = logging.getLogger(logger_name)
+
+    # Log to stdout
+    logger.addHandler(get_stdout_handler())
+    logger.setLevel(loglevel)
+    logger.propagate = False
+    return logger
+
+
+def get_level_name(loglevel):
+    return logging.getLevelName(loglevel)
+
+
+def add_access_logger(app, logger):
+    wsgi_log_format_string = ('"%(REQUEST_METHOD)s %(REQUEST_URI)s %(HTTP_VERSION)s" '
+                              '%(status)s %(bytes)s')
+
+    app.wsgi_app = TransLogger(app.wsgi_app, logger_name=logger.name, format=wsgi_log_format_string,
```

Comment:
Is this a trick after switching to cherrypy? We recently created a module called [sesamutils](https://github.com/sesam-community/sesamutils) to simplify some common microservice tasks. This also does a lot of the stuff you're doing here. I'm thinking we should add access log support to that module if this is a common pattern. If you have time, feel free to add a PR!

Reply:
yes, this is the way.  see the slack conversation [here](https://bouvet.slack.com/archives/G53AH9YTA/p1557391506020500)
will check sesamutils later on for sure.